app/models/ScmemeVariance.scala

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
The file name is mis-named "ScmemeVariance", but also I think a better name for this would be something like SchemeVarianceLockResult.